components/engine_rocks/src/rocks_metrics.rs

```diff
@@ -905,6 +905,10 @@ pub fn flush_engine_properties(engine: &DB, name: &str, shared_block_cache: bool
             STORE_ENGINE_BLOCK_CACHE_USAGE_GAUGE_VEC
                 .with_label_values(&[name, cf])
                 .set(block_cache_usage as i64);
+            let blob_cache_usage = engine.get_blob_cache_usage_cf(handle);
```

Comment:
blob cache is not shared. let's move it out of `if !shared_block_cache {...}`

Reply:
Fixed, I misunderstood this parameter.